EIPS/eip-5089.md

```diff
@@ -0,0 +1,340 @@
+---
+eip: 5089
+title: Principal Token Standard
+description: A standard for principal tokens (zero-coupon tokens) that are redeemable for a single underlying ERC-20 token at a future timestamp.
+author: Julian Traversa (@JTraversa), Robert Robbins (@robrobbins)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-principal-token-standard/1
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-05-01
+requires: 20, 2612
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard API for principal tokens --
+representing ownership of an underlying [ERC-20](./eip-20.md) token at a future timestamp.
+This standard is an extension on the ERC-20 token that provides basic functionality for depositing
+and withdrawing tokens and reading balances and the ERC-2612 standard that provides
+EIP-712 signature based approvals.
+
+## Motivation
+
+Principal tokens have an increasing lack of standardization which has led to a difficult to navigate development space and diverse implementation
+schemes.
+
+The primary examples include yield tokenization platforms which strip future yield leaving a principal
+token behind, as well as fixed-rate money-markets which utilize principal tokens as a medium
+to lend/borrow.
+
+This inconsistency in implementation makes integration difficult at the application layer as well as
+wallet layer which are key catalysts for the space's growth. 
+Developers are currently expected to implement individual adapters for each principal token, as well as adapters for
+their pool contracts, and many times adapters for their custodial contracts as well, wasting significant developer resources. 
+
+## Specification
+
+All ERC-5070 Principal Tokens MUST implement ERC-20 to represent ownership of future underlying redemption (PTs).
```

Comment:
Wrong EIP number, and incorrect abbreviation placement?

```suggestion
Each ERC-5089 Principal Token (PT) MUST implement ERC-20 to represent ownership of future underlying redemption.
```

Reply:
Fixed